src/SmiServices/Microservices/CohortExtractor/ProjectPathResolvers/ProjectPathResolverFactory.cs

```diff
@@ -0,0 +1,17 @@
+using System;
+using System.IO.Abstractions;
+
+namespace SmiServices.Microservices.CohortExtractor.ProjectPathResolvers;
+internal class ProjectPathResolverFactory
+{
+    public static IProjectPathResolver Create(string projectPathResolverType, IFileSystem fileSystem)
+    {
+        return projectPathResolverType switch
+        {
+            "StudySeriesSOPProjectPathResolver" => new StudySeriesSOPProjectPathResolver(fileSystem),
```

Comment:
Better to use `nameof(StudySeriesSOPProjectPathResolver)` rather than a bare string here?